### PR TITLE
Cool markdown popups

### LINKED ISSUE #376: Font size and gutter theme
Hello,

I am having a problem with the font size of the popups, here is a screenshot:

Imgur

The font is really too big, and I can't find any settings related to this problem.

Also, I am having a problem with the gutter theme. I correctly set the gutter theme of my choice in my settings but it doesn't seem to load. Here is a screenshot:

Imgur

The warning icon stays the same whatever the gutter theme I use.

Thank you very much in advance for any help

### PR DESCRIPTION
**Pull request by niosus**
This PR uses the cool unified popups from here: https://github.com/facelessuser/sublime-markdown-popups

This allows to create popups with markdown. This in turn allows code with syntax highlighting etc.
This should fix #376 

---

# Requirements #
- [x] Reference an issue in this PR. Create new one if needed.
- [x] Make sure your branch is based on `dev`.

### CODE REVIEW

File: EasyClangComplete.py [hunk 1]
```diff
@@ -402,19 +402,19 @@ def info_finished(self, future):
             return
         if future.cancelled():
             return
-        (tooltip_request, result) = future.result()
-        if result == "":
+        (tooltip_request, current_popup) = future.result()
+        if current_popup == "":
```
Comment (niosus) [inline EasyClangComplete.py]: Note to self. Change this.

**Comment by niosus** [comment]
Hey @kjteske I am moving away from the old version of showing popups and am bumping into an issue that I don't know how to translate the objective c part to the new version. Are you still using the plugin? If yes, could you please have a look at the previous commit 0d066845419594127c154045a1f418bbdd02ab3b and test out this? If no, I guess I would just remove the objective c support from the plugin alltogether quite soon as it is hard to support a feature that I have no way to test myself.

**Comment by kjteske** [comment]
Yes, I still use and rely on this plugin quite a bit. I'll try to take a look by the end of this weekend and submit a PR. Thanks for the heads up!

**Comment by niosus** [comment]
Great to hear this. I don't often hear about the things when they work :) I will try to submit an untested version of new info message for objective c then and will ping you when I merge it for you to test it if that's ok. Thanks for a fast reply.

File: plugin/popups/popups.py [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        """
+        if extent.start.file.name != extent.end.file.name:
+            return None
+
+        with open(extent.start.file.name, 'r') as f:
+            lines = f.readlines()
+            return "".join(lines[extent.start.line - 1:extent.end.line])
+
+    @staticmethod
+    def info_objc(cursor):
```
Comment (niosus) [inline plugin/popups/popups.py]: @kjteske this is a new method for creating an info popup for objective c. I tried my best to keep the old structure and I feel like it should work out of the box. However, this requires a real world test before being pushed to everyone.

Comment (kjteske) [reply plugin/popups/popups.py]: This looks good to me after playing around with it for a few minutes. I think it's ready to push. I'll keep using it and put up a PR if I notice any problems. Thanks for keeping the obj-c support!

Comment (kjteske) [reply plugin/popups/popups.py]: I'll see if I can add useful tests to test_errors_vis.py this weekend.

Comment (niosus) [reply plugin/popups/popups.py]: No rush. If this works for now take your time. I just want to be sure I can keep supporting this without bugging you every time I need to touch this code.

**Pull request merged**